xep-0045.xml

```diff
@@ -1385,6 +1385,7 @@
 </presence>
 ]]></example>
       <p>Before attempting to enter the room, a MUC-compliant client SHOULD first discover its reserved room nickname (if any) by following the protocol defined in the <link url='#reservednick'>Discovering Reserved Room Nickname</link> section of this document.</p>
+      <p>When a MUC service receives an &lt;x/&gt; tagged join stanza from an already-joined client (as identified by the client's full JID), the service SHOULD assume that the client lost its synchronization, and therefore it SHOULD send exactly the same stanzas to the client as if it would actually join the MUC. The server MAY also send a presence update to the other participants according to the received join presence.</p>
```

Comment:
> strike this and just say "the service SHOULD send…"

Having rationale for design decisions is a very important part of a good specification. I understand that the placement of "SHOULD" is not perfect here, and I'm open for better wording suggestions that don't remove the rationale.

> [...] needs to be rephrased

Would "as if it actually joined the MUC" be a better wording? I'm not a native speaker, and I struggled a bit with writing this. Again, I'm open for suggestions.